src/zenml/cli/metadata_store.py

```diff
@@ -84,6 +88,40 @@ def list_metadata_stores() -> None:
     )
 
 
+@metadata_store.command(
+    "describe",
+    help="Show details about the current active metadata store.",
+)
+@click.argument(
+    "metadata_store_name",
+    type=click.STRING,
+    required=False,
+    default="",
+)
+def describe_metadata_store(metadata_store_name: str) -> None:
+    """Show details about the current active metadata store."""
+    repo = Repository()
+    if metadata_store_name == "":
+        metadata_store_name = repo.get_active_stack().metadata_store_name
+
+    metadata_stores = repo.get_service().metadata_stores
+    if len(metadata_stores) == 0:
+        cli_utils.warning("No metadata stores registered!")
+        return
+
+    try:
+        metadata_store_details = metadata_stores[metadata_store_name]
+    except KeyError:
+        cli_utils.error(
+            f"Metadata store `{metadata_store_name}` does not exist."
+        )
+        return
+    cli_utils.title("Active Metadata Store:")
+    cli_utils.declare(f"NAME: {metadata_store_name}")
+    cli_utils.declare(f"UUID: {metadata_store_details.uuid}")
+    cli_utils.declare(f"URI: {metadata_store_details.uri}")  # type: ignore[attr-defined] # noqa
```

Comment:
The problem with this is that not every metadata store has a `uri` attribute (e.g. the MySQL one has host, port, ...).
I think we need to use a dynamic way to calculate what we print here using the actual attributes available for this specific metadata store (equivalent for all other stack components).

I think we can use pydantics `dict()` method to achieve at least a reasonable first implementation:
```python
for key, value in metadata_store.dict().items():
    cli_utils.declare(f"{key.upper()": {value}")
```

Reply:
Also it prints "Active Metadata Store" even if I specify a name for a metadata store that is not active, maybe the title should just be the name of the store instead? And maybe a `cli_utils.declare()` to print whether its active or not?

Reply:
I think your comments also align with my review. We need a way to generate this describe dynamically just like kubectl does it